openwisp_controller/config/tests/test_admin.py

```diff
@@ -1626,89 +1567,6 @@ def test_add_vpn(self):
             response, 'value="openwisp_controller.vpn_backends.OpenVpn" selected'
         )
 
-    def test_vpn_clients_deleted(self):
```

Comment:
why are these tests being moved?

Reply:
As we are adding the templates via js so i migrated these tests to selenium tests. I have added this in conversation above. I will update the description of PR itself for more clarity.